src/nl/hannahsten/texifyidea/lang/commands/LatexDiffcoeffCommand.kt

```diff
@@ -0,0 +1,38 @@
+package nl.hannahsten.texifyidea.lang.commands
+
+import nl.hannahsten.texifyidea.lang.LatexPackage
+import nl.hannahsten.texifyidea.lang.LatexPackage.Companion.SIUNITX
+
+/**
+ * @author Hannah Schellekens
+ */
+enum class LatexSiunitxCommand(
+    override val command: String,
+    override vararg val arguments: Argument = emptyArray(),
+    override val dependency: LatexPackage = LatexPackage.DEFAULT,
+    override val display: String? = null,
+    override val isMathMode: Boolean = true,
+    val collapse: Boolean = false
+) : LatexCommand {
+
+    DIFF("diff", "order-spec".asOptional(), "differentiand".asRequired(), "variable(s)".asRequired(), "pt of eval".asOptional(), dependency = DIFFCOEFF),
```

Comment:
Usually, commands should be added to the autocompletion automatically, and these files are only necessary if we need to hardcode more information, like which greek command produces which symbol.
However, I checked the source of this package and it looks like commands are defined like `\DeclareInstance { diffcoeff } { difsp } { DIFSP } {}`, I didn't know about templates like this but it seems it is used more often so I think we should do this generically.
Now that you have this file already we can just as well include it like this though